lib/backtrace_cleaner.rb

```diff
@@ -0,0 +1,26 @@
+module Fiverr
+  class BacktraceCleaner < ActiveSupport::BacktraceCleaner
+    APP_DIRS_PATTERN        = /^\/?(app|config|lib|test)/
+    FIVERR_GEMS_PATTERN     = /.*bundler\/gems/
+    RENDER_TEMPLATE_PATTERN = /:in `_render_template_\w*'/
+
+    def initialize
+      super
+      add_filter   { |line| line.sub("#{Rails.root}/", '') }
```

Comment:
I assume add_filter is implemented in base class?

Reply:
you assume right